faiss/IndexRefine.cpp

```diff
@@ -54,6 +54,14 @@ void IndexRefine::add(idx_t n, const float* x) {
     ntotal = refine_index->ntotal;
 }
 
+size_t IndexRefine::remove_ids(const IDSelector& sel) {
+    FAISS_THROW_IF_NOT(is_trained);
+    base_index->remove_ids(sel);
+    refine_index->remove_ids(sel);
+    ntotal = refine_index->ntotal;
+    return -1;
```

Comment:
I believe the return value is supposed to be the number of vectors removed. You should be able to do this by getting the ntotal of the refine_index (or base_index - they should be the same) before applying the remove_ids operation.

Reply:
sorry, I will fix it